packages/toggleContent/components/toggleContent.tsx

```diff
@@ -45,7 +47,7 @@ export class ToggleContent extends React.PureComponent<
 
     return (
       <Clickable tabIndex={tabIndex} action={this.handleClick}>
-        <div className={style}>{content}</div>
+        <span className={style}>{content}</span>
```

Comment:
This will cause the tests in DCOS UI to fail but we can update the snapshot with ease.
https://github.com/dcos/dcos-ui/pull/3218/files#diff-b344e4851c4356add413f1168ead98a5R75

Reply:
Updated the snaps already